lib/isaac/isaac_kernel.hpp

```diff
@@ -757,7 +757,9 @@ namespace isaac
                             normal = start + t0 * dir - particle_pos;
                             if( t0 < 0 && is_clipped )
                             {
-                                color.w = 0;
+                                #if ISAAC_AO_BUG_FIX == 1
```

Comment:
I realized now that this will always be false because you never passed a define `ISAAC_AO_BUG_FIX ` to the compiler.
There is only a cmake option which is doing nothing.

Reply:
Good catch - sorry for the fault.